sharpy/utils/datastructures.py

```diff
@@ -165,13 +165,15 @@ def __init__(self, dimensions, dimensions_star):
                                                dimensions_star[i_surf, 1] + 1),
                                                dtype=ct.c_double))
 
-        # total forces
+        # total forces - written by AeroForcesCalculator
         self.inertial_total_forces = np.zeros((self.n_surf, 6))
         self.body_total_forces = np.zeros((self.n_surf, 6))
         self.inertial_steady_forces = np.zeros((self.n_surf, 6))
         self.body_steady_forces = np.zeros((self.n_surf, 6))
         self.inertial_unsteady_forces = np.zeros((self.n_surf, 6))
         self.body_unsteady_forces = np.zeros((self.n_surf, 6))
+        self.total_inertial_forces = np.zeros((6,))  # G Frame
+        self.total_body_forces = np.zeros((6,))  # A Frame
```

Comment:
These forces are not used anywhere as well are others. You committed the necessary changes in commit 8d1021e30e195a79795c6ed8357c1ac286367e0b